modules/tinymce/src/plugins/directionality/test/ts/browser/CommandsTest.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/directionality/Plugin';
+
+describe('browser.tinymce.plugins.directionality.CommandsTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    plugins: 'directionality',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ], true);
+
+  const withNoneditableRootEditor = (editor: Editor, f: (editor: Editor) => void) => {
```

Comment:
as a reminder, it is possible to remove it if `feature/TINY-9669` will be merged before this